sixpack.js

```diff
@@ -199,7 +199,21 @@
         } else {
             var httpModule = url.startsWith('https') ? 'https' : 'http';
             var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
-            var req = http.get(url, { headers: { 'Cookie': cookie } }, function(res) {
+
+            const parsedUrl = new URL(url);
+            const options = {
+                protocol: parsedUrl.protocol,
+                host: parsedUrl.host,
+                port: parsedUrl.port,
+                hostname: parsedUrl.hostname,
+                search: parsedUrl.search,
+                pathname: parsedUrl.pathname,
+                path: parsedUrl.pathname + parsedUrl.search, 
+                href: parsedUrl.href,
+                headers: { Cookie: cookie }
+            }
```

Comment:
I think you could use `Object.assign()`

Reply:
I think `new URL(url)` object has no enumerable properties. `Object.assign` works only with enumerable object own properties.